ui/src/groups/AddGroupSheet.tsx

```diff
@@ -0,0 +1,87 @@
+import { useState } from 'react';
+import cn from 'classnames';
+import WidgetDrawer from '@/components/WidgetDrawer';
+import HomeIconMobileNav from '@/components/icons/HomeIconMobileNav';
+import NewRaysIcon from '@/components/icons/NewRaysIcon';
+import { isNativeApp } from '@/logic/native';
+import JoinGroup from './AddGroupSheet/JoinGroup';
+import CreateGroup from './AddGroupSheet/CreateGroup';
+
+function CreateOrJoin(props: { onCreate: () => void; onJoin: () => void }) {
+  return (
+    <div className="flex w-full flex-col items-center pb-4">
+      <h3 className="mb-6 mt-4 text-[17px] ">Add a group</h3>
+      <div className="flex flex-col rounded-lg border border-gray-100">
```

Comment:
Should have `overflow-hidden` to ensure the background colors are rounded as well:

<img width="366" alt="Screenshot 2024-01-15 at 4 37 12 PM" src="https://github.com/tloncorp/landscape-apps/assets/1013230/1eca3861-c727-4d93-8509-33373729ede4">